www/js/con-author.js

```diff
@@ -24,11 +24,12 @@ define([
 	'dojo/dom',
 	'dojo/dom-class',
 	'dijit/registry',
+	'dojo/NodeList-dom',
 	'./controller',
 	"./equation",
 	"./typechecker",
 	"dojo/domReady!",
-], function(array, declare, lang, style, keys, ready, on, memory, aspect, dom, domClass, registry, controller, equation, typechecker){
+], function(array, declare, lang, style, keys, ready, on, memory, aspect, dom, domClass, registry, domList, controller, equation, typechecker){
```

Comment:
domList is the return object of dojo/query. It is used for defining forEach that I use.